user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Button.md

```diff
@@ -29,6 +29,9 @@ To configure the component:
 
 In the Dashboards app, the soft-launch button component can be used to execute the action of a button parameter. As such, it must be configured with button parameter data.
 
+> [!NOTE]
+> If the *ReportsAndDashboardsAutomationScript* soft-launch option is enabled as well, this component functions as an Automation script component, and you can link it directly to an Automation script in the *Settings* pane.
```

Comment:
the button functions like an automation script component until you add the button parameter, then it acts as a parameter button

Reply:
```suggestion
> If the *ReportsAndDashboardsAutomationScript* soft-launch option is enabled as well, this component functions as an Automation script component, and you can link it directly to an Automation script in the *Settings* pane. However, if you add a button parameter to the component, it will still act as a parameter button even with this soft-launch option.
```